specs/gcpresource.spec

```
@@ -0,0 +1,90 @@
+# Model
+model:
+  rest_name: gcpresource
+  resource_name: gcpresources
+  entity_name: GCPResource
+  package: pandemona
+  group: pcn/infrastructure
+  description: Represents a GCP cloud resource such as a virtual machine.
+  extends:
+  - '@identifiable-stored'
+  - '@migratable'
+  - '@namespaced'
+  - '@zoned'
+
+# Indexes
+indexes:
+- - namespace
+  - selflink
+- - namespace
+  - nativeID
+
+# Attributes
+attributes:
+  v1:
+  - name: RRN
+    description: Prisma Cloud RRN.
+    type: string
+    exposed: true
+    omit_empty: true
+
+  - name: data
+    description: The JSON-encoded data that represents the resource.
+    type: external
+    exposed: true
+    subtype: '[]byte'
+    stored: true
+    required: true
+    example_value: |-
+      {
+        "id": "12345",
+        "kind": "compute#instance"
+        ...
+      }
+
+  - name: kind
+    description: The specific kind of the resource.
+    type: enum
+    exposed: true
+    stored: true
+    required: true
+    allowed_choices:
+    - Instance
+    - Subnet
+    - VirtualNetwork
```

let's not change the terminologies and use gcp terms and acronyms. Also, let's prefix them with their domains.. so if we run into something like `database#instance`, we don't shit our pants :)
```suggestion
    - ComputeInstance
    - ComputeSubnetwork
    - ComputeNetwork
```